python/dgl/contrib/data/knowledge_graph.py

```diff
@@ -327,7 +327,7 @@ def _load_data(dataset_str='aifb', dataset_path=None):
     train_file = os.path.join(dataset_path, 'trainingSet.tsv')
     test_file = os.path.join(dataset_path, 'testSet.tsv')
     if dataset_str == 'am':
-        label_header = 'label_category'
+        label_header = 'label_cateogory'
```

Comment:
revert

Reply:
This is needed to run AM. The header in the dataset is actually misspelled.

Reply:
Ugh.